docs/learn/rossum-formulas/formula-fields.md

```diff
@@ -17,6 +17,13 @@ For most tasks, Rossum’s Copilot handles everything seamlessly without needing
 This powerful feature is available on the Business plan and above. Existing customers interested in using formula fields can reach out to our support team at support@rossum.ai for assistance.
 
 :::
+## Basic information
+- FFs can run any Python code including its [Standard Library modules.](https://docs.python.org/3/library/index.html) 
```

Comment:
```suggestion
- FFs can run any Python code including its [Standard Library modules](https://docs.python.org/3/library/index.html).
```

Also @VaclavRut, can you please avoid using shortcuts? We do not explain what "FF" stands for anywhere on the whole website (even though it is quite obvious to us internally).